example/mnist/mnist/custom_evaluator.py

```diff
@@ -166,19 +169,28 @@ def _load_model(self, device: torch.device) -> Net:
         print("load mnist model, start to inference...")
         return model
 
-    def _pre(self, input: Image) -> torch.Tensor:
-        _tensor = torch.tensor(bytearray(input.to_bytes()), dtype=torch.uint8).reshape(
-            input.shape[0], input.shape[1]  # type: ignore
-        )
-        _image_array = PILImage.fromarray(_tensor.numpy())
-        _image = transforms.Compose(
-            [transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))]
-        )(_image_array)
-        return torch.stack([_image]).to(self.device)
+    def _pre(self, data: t.List[Image]) -> t.Any:
+        images = []
+        for i in data:
```

Comment:
Can we use the torch array operation to replace the `for-item-append` operation?

Reply:
this has been rollbacked to single